src/table/hooks/use-snapshot.ts

```diff
@@ -1,4 +1,5 @@
-import { onTakeSnapshot } from '@nebula.js/stardust';
+// @ts-ignore ignore useImperativeHandle
```

Comment:
ts-expect-error is a better alternative in this case

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error